SimTracker/TrackerHitAssociation/plugins/ClusterSLOnGPU.cu

```diff
@@ -35,7 +33,7 @@ void simLink(const SiPixelDigisCUDA::DeviceConstView *dd, uint32_t ndigis, const
 
   auto ch = pixelgpudetails::pixelToChannel(dd->xx(i), dd->yy(i));
   auto first = hh.hitsModuleStart_d[id];
-  auto cl = first + cc->clus(i);
+  auto cl = first + dd->clus(i);
```

Comment:
I do not understand this change, so I trust that it is fine...

Reply:
It is a consequence of moving `clus` from `SiPixelClustersCUDA` to `SiPixelDigisCUDA` which is the proper place for that array (which I didn't realize in #172).